charts/karmada/_crds/kustomization.yaml

```diff
@@ -18,6 +18,8 @@ resources:
 - bases/remedy/remedy.karmada.io_remedies.yaml
 - bases/apps/apps.karmada.io_workloadrebalancers.yaml
 
-patchesStrategicMerge:
-- patches/webhook_in_resourcebindings.yaml
-- patches/webhook_in_clusterresourcebindings.yaml
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
```

Comment:
Is it okay if these two lines are not placed at the top of the file?

Reply:
it's generate by kustomize, i will suggesstion to keep it for consistent with kustomize.

now i Removed it for resolve comment.

Reply:
Aren't these two lines necessary?

Reply:
Thanks, I get it. 

/lgtm
/approve

Reply:
Thanks you!